### ISSUE TITLE
AppLovinPrivacySettings error on android

### ISSUE THREAD

**Issue by mhanzla80** [ioridev/flutter_applovin_max#52]
### AppLovinPrivacySettings error on android

**Output:**
`/Users/hanzla/Development/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_applovin_max-2.1.2/android/src/main/java/dev/iori/flutter_applovin_max/FlutterApplovinMaxPlugin.java:80: error: cannot find symbol
                    AppLovinPrivacySettings.setHasUserConsent( call.argument("HasConsent"), context );
                    ^
  symbol:   variable AppLovinPrivacySettings
  location: class FlutterApplovinMaxPlugin
/Users/hanzla/Development/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_applovin_max-2.1.2/android/src/main/java/dev/iori/flutter_applovin_max/FlutterApplovinMaxPlugin.java:81: error: cannot find symbol
                    AppLovinPrivacySettings.setIsAgeRestrictedUser( call.argument("IsAgeRestricted"), context );
                    ^
  symbol:   variable AppLovinPrivacySettings
  location: class FlutterApplovinMaxPlugin
/Users/hanzla/Development/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_applovin_max-2.1.2/android/src/main/java/dev/iori/flutter_applovin_max/FlutterApplovinMaxPlugin.java:82: error: cannot find symbol
                    AppLovinPrivacySettings.setDoNotSell( call.argument("DoNotSell"), context );
                    ^
  symbol:   variable AppLovinPrivacySettings
  location: class FlutterApplovinMaxPlugin
Note: /Users/hanzla/Development/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_applovin_max-2.1.2/android/src/main/java/dev/iori/flutter_applovin_max/FlutterApplovinMaxPlugin.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
3 errors

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':flutter_applovin_max:compileDebugJavaWithJavac'.
> Compilation failed; see the compiler error output for details.
`

** AppLovinPrivacySettings throwing error on android but working fine on iOS**
How do we have to solve this ?

**Comment by andalasid** [comment]
yes, I've got this issue too. Don't know how this happened. please help to fix this.

**Comment by Amit9071** [comment]
yes, I've got this issue too. anyone know solution how to resolve?

**Comment by thomasmso** [comment]
Feel free to migrate over to the official plugin: https://github.com/AppLovin/AppLovin-MAX-Flutter

**Comment by Amit9071** [comment]
Thanks @thomasmso for your response.
I used thhis official plugin. but i am facing one issue like when i use banner image it's not showing in column. it's showing on screen like top,bottom and center. i want to use in column. so you have idea?

**Comment by thomasmso** [comment]
Sorry, but we don't support displaying banner vertically.

**Comment by imyogisingh** [comment]
Fix this issue please 🙏 . The official plugin is not working properly.

**Comment by anandyadav21219** [comment]
Facing the same issue.

**Comment by anandyadav21219** [comment]
I have fixed the error by doing the below steps:

- "flutter clean"
- "flutter pub get"
- Open the android folder in the new android studio window
- Now run the app and you will  get an error
- I just rich out the line at error
- you just need to change three lines of code(cross-check with the below image)
- or try out the suggestion. it's the same as in flutter.
- and I am good to go running the app.

I think this one is a temporary solution but working for me.

![image](https://user-images.githubusercontent.com/54627691/193410863-7e0b1acc-acc7-49cf-8b68-d595f73abea4.png)
